corehq/form_processor/models.py

```diff
@@ -563,6 +563,11 @@ def is_deleted(self):
     def dynamic_case_properties(self):
         return OrderedDict(sorted(self.case_json.iteritems()))
 
+    def get_json(self, lite=False):
```

Comment:
I realize you didn't pick the names, but `get_json` and `to_json` are pretty confusing to reason with in terms of what the expected differences should be. Could we potentially change this to `get_api_json` or something?